package.json

```diff
@@ -40,16 +40,16 @@
     "flow-bin": "^0.118.0",
     "flow-typed": "^2.6.2",
     "html-loader": "^1.0.0",
-    "html-webpack-plugin": "^4.0.3",
+    "html-webpack-plugin": "^5.3.1",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^25.3.0",
     "redux-logger": "^3.0.6",
     "redux-mock-store": "^1.5.4",
     "style-loader": "^1.1.3",
-    "webpack": "^4.41.5",
+    "webpack": "^5.37.0",
     "webpack-bundle-analyzer": "^3.9.0",
-    "webpack-cli": "^3.3.10",
-    "webpack-dev-server": "^3.10.3",
+    "webpack-cli": "^4.7.0",
+    "webpack-dev-server": "^3.11.2",
```

Comment:
For an enhanced dev experience with live reloading you might want to try out webpack dev server version ^4.0.0-beta.3 and "@pmmmwh/react-refresh-webpack-plugin": "^0.5.0-beta.8". I ve seen that in some conditions native live reloading of webpack 5 is not kicked in sometimes.